colossalai/nn/optimizer/cpu_adamw.py

```diff
@@ -0,0 +1,124 @@
+import torch
+import math
+
+
+class CPUAdamW(torch.optim.Optimizer):
+    optimizer_id = 0
+
```

Comment:
Can we set `num_fp32_shards_per_param` to support hybrid optimization?

Reply:
OK, I forget to update the code.